src/test/client-test.cc

```diff
@@ -47,6 +47,10 @@ using hbase::Table;
 using hbase::TestUtil;
 
 class ClientTest : public ::testing::Test {
+ protected:
+  ~ClientTest() override {
```

Comment:
Its a long time... whats the '~' do?

Reply:
It's the C++ destructor; I put this in to make sure the first few tests didn't pollute the remaining ones but on reflection I probably should have overridden the proper `TearDownTestCase` method in googletest.

Reply:
Duh. Thanks.